awx/urls.py

```diff
@@ -6,11 +6,14 @@
 
 from awx.main.views import handle_400, handle_403, handle_404, handle_500, handle_csp_violation, handle_login_redirect
 
+from ansible_base.lib.dynamic_config.dynamic_urls import api_version_urls as base_urls
+
 
 urlpatterns = [
     re_path(r'', include('awx.ui.urls', namespace='ui')),
     re_path(r'^ui_next/.*', include('awx.ui_next.urls', namespace='ui_next')),
     re_path(r'^api/', include('awx.api.urls', namespace='api')),
+    re_path(r'^sso/', include(base_urls)),
     re_path(r'^sso/', include('awx.sso.urls', namespace='sso')),
```

Comment:
Do we need to remove the `'awx.sso.urls'` entry?

Reply:
we're still using the other backends in sso so I've left the urls